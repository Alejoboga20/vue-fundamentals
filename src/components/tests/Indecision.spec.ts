import { VueWrapper, shallowMount } from '@vue/test-utils';
import Indecision from '../Indecision.vue';

describe('Indecision.vue', () => {
	let wrapper: VueWrapper<any>;

	beforeEach(() => {
		wrapper = shallowMount(Indecision);
	});

	test('should match the snapshot', () => {
		expect(wrapper.html()).toMatchSnapshot();
	});
});
