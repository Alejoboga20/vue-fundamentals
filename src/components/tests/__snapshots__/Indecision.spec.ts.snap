// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Indecision.vue > should match the snapshot 1`] = `
"<!--v-if-->
<div data-v-4a8fca50=\\"\\" class=\\"bg-dark\\"></div>
<div data-v-4a8fca50=\\"\\" class=\\"indecision-container\\"><input data-v-4a8fca50=\\"\\" type=\\"text\\" placeholder=\\"ask me a question\\">
  <p data-v-4a8fca50=\\"\\">End your question with a question (?) mark</p>
  <div data-v-4a8fca50=\\"\\" style=\\"display: none;\\">
    <h2 data-v-4a8fca50=\\"\\"></h2>
    <h1 data-v-4a8fca50=\\"\\"></h1>
  </div>
</div>"
`;
